### PR TITLE
Updates of release 2019-05

### PR DESCRIPTION
**Pull request by herflis**

### CODE REVIEW

File: _updates/2019-05-02-may-2019.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: May 2019
+---
+
+## Mononorepo changes
+- All packages build now with project references in packages folder (yarn build now runs tsc -b packages)
+  
+## Packages
+
+### [@sensenet/client-core@1.6.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fclient-core%401.6.0)
+- Passing request inits to fetch
```
Comment (zoltanbedi) [inline _updates/2019-05-02-may-2019.md:11]: There was a breaking change in client core. see https://github.com/SenseNet/sn-client/pull/152

File: _updates/2019-05-02-may-2019.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- Class for BooleanFieldSetting is added
+- SelectionRoots prop to NullFieldSetting is added because Avatar is a NullFieldSetting by default in the schema
+  
+### [@sensenet/repository-events@1.4.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Frepository-events%401.4.0)
+- onUpload change event improvements
+
+### [@sensenet/controls-react@2.6.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fcontrols-react%402.6.0)
+- Boolean fieldcontrol is added
+- ShortText and Password fields' new view is fixed
+- Move assets outside of src
```
Comment (zoltanbedi) [inline _updates/2019-05-02-may-2019.md]:
```suggestion
```
in this package the assets were completely removed.

File: _updates/2019-05-02-may-2019.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- redux related things (logic, dependencies, etc) are completely removed from the package
+- Forminput hint text style improvements
+
+### [@sensenet/list-controls-react@1.4.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Flist-controls-react%401.4.0)
```
Comment (zoltanbedi) [inline _updates/2019-05-02-may-2019.md]: This should be pickers react.

File: _updates/2019-05-02-may-2019.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## Mononorepo changes
+- All packages build now with project references in packages folder (yarn build now runs tsc -b packages)
+  
+## Packages
+
+### [@sensenet/client-core@2.0.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fclient-core%402.0.0)
+- Passing request inits to fetch
+
```
Comment (gallayl) [inline _updates/2019-05-02-may-2019.md:13]:
```suggestion
 - refactored upload and made it available in the repository object

```

File: _updates/2019-05-02-may-2019.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Renaming a content is fixed
+- 'Add to group' and 'delete user' buttons are now active only when at least one user is selected
+- 'Add user' button is now active only when at least one group is selected
+- Mobile drawer menu is fixed
+
+### [@sensenet/sn-app@0.2.0](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fsn-app%400.2.0)
+
+- Localization
+- Added dirty form checking for Text Editor
+- From redux to context, added save & cancel button for text editor
```
Comment (gallayl) [inline _updates/2019-05-02-may-2019.md]:
```suggestion
- Moved editor-related state from redux to context, added save & cancel button for text editor
```

**Review by gallayl — CHANGES_REQUESTED** [review]

**Pull request merged**